### PR TITLE
added contract and deploy script

### PR DESCRIPTION
**Pull request by mishra811**
I have added contracts and deploy scripts under `smart_contracts` directory. Please review the contract and let me know of any other features to be added to the contract.

### CODE REVIEW

File: smart_contracts/test/Lock.js [hunk 1]
```diff
@@ -0,0 +1,126 @@
...
+    const [owner, otherAccount] = await ethers.getSigners();
+
+    const Lock = await ethers.getContractFactory("Lock");
+    const lock = await Lock.deploy(unlockTime, { value: lockedAmount });
+
+    return { lock, unlockTime, lockedAmount, owner, otherAccount };
+  }
+
+  describe("Deployment", function () {
+    it("Should set the right unlockTime", async function () {
```
Comment (felixfaisal) [inline smart_contracts/test/Lock.js:29]: I believe these tests are meant for the vault smart contract right? Let's not have it here

File: smart_contracts/contracts/Web3Forms.sol [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+
+    constructor(){
+        s_owner= msg.sender;
+    }
+
+    struct Form{
+        uint formId;
+        address creator;
+        uint publishedTime;
+        string formData;
```
Comment (felixfaisal) [inline smart_contracts/contracts/Web3Forms.sol:23]: For now we are assuming the form data to be in string?

**Review by felixfaisal — CHANGES_REQUESTED** [review]

File: smart_contracts/test/Lock.js (hunk 1 above)
Comment (mishra811) [reply smart_contracts/test/Lock.js:29]: Yes I will remove it ASAP

File: smart_contracts/contracts/Web3Forms.sol (hunk 2 above)
Comment (mishra811) [reply smart_contracts/contracts/Web3Forms.sol:23]: Yes so for now this `string formData` is representing the IPFS hash which will contain all the data in JSON format

Comment (felixfaisal) [reply smart_contracts/contracts/Web3Forms.sol:23]: Cool, That makes sense ✅

**Review by felixfaisal — APPROVED** [review]

**Pull request merged**